src/domain/zemlja.py

```diff
@@ -20,7 +20,26 @@ class Zemlja:
     hrana: Hrana = None
 
     def __post_init__(self):
-        self.snake = Snake(x=self.sirina / 2, y=self.sirina / 2, velikost=0)
+        self.snake = Snake(x=self.sirina / 2, y=self.visina / 2, dx=-1, dy=0)
         self.hrana = Hrana(
-            x=random.randint(0, self.sirina),
-            y=random.randint(0, self.visina))
+            x=random.randint(1, self.sirina - 1),
+            y=random.randint(1, self.visina - 1))
+
+    def nastavi_hrano(self):
+        self.hrana.x = random.randint(1, self.sirina - 1)
+        self.hrana.y = random.randint(1, self.sirina - 1)
+
+    def konec(self) -> bool:
+        # če je kača v okviru dimenzij zemlje
+        if 0 <= self.snake.x < self.sirina and 0 <= self.snake.y < self.visina:
+            return True
+        else:
+            return False
+
+    def dodaj_del_kace_in_nastavi_hrano(self) -> int:
+        if self.snake.x == self.hrana.x and self.snake.y == self.hrana.y:
+            self.snake.dodaj_del_kace()
+            self.nastavi_hrano()
+            return 1
+        else:
+            return False
```

Comment:
V definiciji funkcije si povedal da vračaš `int` vendar tukaj vračaš `False` in to bi se compiler pritožil če bi ga python imel.